src/components/oscal-utils/OSCALPropUtils.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * When a namespace is not specified for a property, the assumption is that the
+ * namespace is default namespace.
+ */
+const NIST_DEFAULT_NAMESPACE = "https://csrc.nist.gov/ns/oscal";
+
+/**
+ * Return the given namespace, or if undefined, the default namespace.
+ */
+export function namespaceOf(ns) {
+  return ns ?? NIST_DEFAULT_NAMESPACE;
+}
+
+/**
+ * Returns the first matching property in the given list.
+ *
+ * @param props the list of properties to check
+ * @param filter the attributes to match against
+ */
+export default function matchingProp(props, filter) {
+  const ns = namespaceOf(filter.ns);
+  if ((!filter.value && !filter.filter) || (filter.value && filter.filter)) {
+    throw new Error("Exactly one of filter or value must be specified");
+  }
+
+  return props.find(
+    (prop) =>
+      namespaceOf(prop.ns) === ns &&
```

Comment:
```suggestion
  if ((!filter.value && !filter.filter) || (filter.value && filter.filter)) {
    throw new Error("Exactly one of filter or value must be specified");
  }

  return props.find(
    (prop) =>
      namespaceOf(prop.ns) === namespaceOf(filter.ns) &&
```

This makes the whole thing more consistent internally and the variable is never used elsewhere.

Reply:
I was wondering - that makes more sense now.